_data/tools.yml

```diff
@@ -14,7 +14,7 @@ CMV: toolshed.g2.bx.psu.edu/repos/rnateam/cmv/cmv
 Canu: toolshed.g2.bx.psu.edu/repos/bgruening/canu/canu
 CoFold: toolshed.g2.bx.psu.edu/repos/rnateam/cofold/cofold
 Compalignp: toolshed.g2.bx.psu.edu/repos/rnateam/compalignp/compalignp
-crosscontamination_barcode_filter: toolshed.g2.bx.psu.edu/repos/iuc/crosscontamination_barcode_filter
+crosscontamination_barcode_filter: toolshed.g2.bx.psu.edu/repos/iuc/crosscontamination_barcode_filter/crosscontamination_barcode_filter
```

Comment:
In theory `crosscontamination_barcode_filter` should also work and point to the latest version. Does anyone want to try that?

Reply:
This does not appear to work:

> https://usegalaxy.eu/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcrosscontamination_barcode_filter

But this does:

> https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcrosscontamination_barcode_filter%2Fcrosscontamination_barcode_filter

Reply:
What I mean is https://usegalaxy.eu/root?tool_id=crosscontamination_barcode_filter